src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -870,6 +870,29 @@ ExecuteModifyTasksWithoutResults(List *taskList)
 
 
 /*
+ * ExecuteSequentialTasksWithoutResults basically calls ExecuteModifyTasks in a
+ * loop in order to simulate sequential execution of a list of tasks. Useful in
+ * cases where issuing commands in parallel before waiting for results could
+ * result in deadlocks (such as CREATE INDEX CONCURRENTLY).
+ */
+int64
```

Comment:
That would make sense, yes. I copy-pasted from `ExecuteModifyTasksWithoutResults` and didn't realize I'd done that.